src/components/EventManagement/Dashboard/EventDashboard.tsx

```
@@ -73,25 +81,47 @@ const EventDashboard = (props: { eventId: string }): JSX.Element => {
+
+  // Extract time from startAt/endAt if needed (assuming ISO format like "2025-04-01T10:00:00Z")
+  const formatTimeFromDateTime = (dateTime: string): string => {
+    if (!dateTime) return '08:00';
+
+    try {
+      const date = new Date(dateTime);
+      const hours = date.getUTCHours().toString().padStart(2, '0');
+      const minutes = date.getUTCMinutes().toString().padStart(2, '0');
+      // Return time in UTC to match the test expectations
+      return `${hours}:${minutes}`;
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)